src/ecs/components/audio/audio_listener.rs

```diff
@@ -0,0 +1,13 @@
+use ecs::{Component, VecStorage};
+
+/// An audio listener, add this component to the local player character.
+pub struct AudioListener {
+    /// Position of the left_ear in the world.
+    pub left_ear: [f32; 3],
+    /// Position of the right ear in the world.
+    pub right_ear: [f32; 3],
+}
+
+impl Component for AudioListener {
+    type Storage = VecStorage<Self>;
```

Comment:
Alright, I'll check out the other storage types and see if I can find a better one.